dependency/actions/compile/bionic.Dockerfile

```diff
@@ -1,73 +1,65 @@
 FROM ubuntu:18.04
```

Comment:
Per https://github.com/paketo-buildpacks/php-dist/issues/486, can we compile on our stack image? Same for Jammy